### PR TITLE
Language links

### PR DESCRIPTION
**Pull request by wytrych**
* Add a directive to generate links instead of a `select`
* Change template naming
* Namespace directives and templates

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js Please review

File: app/scripts/src/selector.js [hunk 1]
```diff
@@ -16,21 +16,33 @@ module.controller('languageSelectorController', [
...
+        };
+    },
+]);
+
+module.directive('incLanguageLinks', [
+    function () {
+        return {
+            restrict: 'A',
+            templateUrl: 'templates/inc-language-select/language-switch/language-links.html',
             scope: {},
```
Comment (henrahmagix) [inline app/scripts/src/selector.js:46]: Does this need `scope: {}`? Is not a new scope made by the controller when injecting `$scope`?

Comment (wytrych) [reply app/scripts/src/selector.js:46]: The controller doesn't have control over the scope, only the directive has

Comment (henrahmagix) [reply app/scripts/src/selector.js:46]: 👍

**Pull request merged**